lib/board.rb

```diff
@@ -0,0 +1,81 @@
+
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+    reset!
+  end
+  def cells
+    @cells
+  end
+
+  def reset!
+
+     @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+
+  end
+
+
+
+
+  def display
+    puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    puts "-----------"
+    puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    puts "-----------"
+    puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+  end
+
+  def position(pos)
+    current_pos = pos.to_i
+
+    cells[current_pos-1]
```

Comment:
lets simplify this into one line. 

pos is a string but we can `pos.to_i - 1` on line 33 instead of creating a new variable that stores pos as an integer